kyo-core/shared/src/test/scala/kyoTest/internal/PositionTest.scala

```diff
@@ -0,0 +1,37 @@
+package kyoTest.internal
+
+import kyo.internal.Position
+import kyoTest.KyoTest
+
+class PositionTest extends KyoTest:
+    def makeErrorMessage(msg: => String)(using f: Position): String =
+        s"$f - $msg"
+
+    "implicitly" in {
+        val f = implicitly[Position]
+        assert(f.toString() == "PositionTest.scala:11 - val f")
```

Comment:
I would prefer to always inline the class/namespace as the owner, not the `val`.

Reply:
That typically matches the file name so I'm not sure it's very useful. The goal is to help the user find the log call, I think the closest it is to it, the better.